automation/repeated_test.sh

```diff
@@ -196,7 +196,35 @@ export KUBEVIRT_DEPLOY_NET_BINDING_CNI=true
 
 export KUBEVIRT_PROVIDER="${TEST_LANE}"
 
-ginko_params="$ginko_params -no-color -succinct --label-filter=(!QUARANTINE)&&(!exclude-native-ssh)&&(!no-flake-check) -randomize-all"
+# add_to_label_filter appends the given label and separator to
+# $label_filter which is passed to Ginkgo --filter-label flag.
+# How to use:
+# - Run tests with label
+#     add_to_label_filter '(mylabel)' ','
+# - Dont run tests with label:
+#     add_to_label_filter '(!mylabel)' '&&'
+add_to_label_filter() {
```

Comment:
I might be missing something but I see this function is also defined (with same code and comments) in automation/test.sh did you intend to delete it from there or why do you need this duplication?

Reply:
yeah I don't know what's the policy around sharing bash functions, I don't think it's a big deal